prow/plugins/lgtm/lgtm.go

```diff
@@ -23,13 +23,23 @@ import (
 	"k8s.io/test-infra/prow/github"
 )
 
+const PluginName = "lgtm"
+
 var (
+	lgtmLabel    = "lgtm"
 	lgtmRe       = regexp.MustCompile(`(?mi)^\/lgtm\r?$`)
 	lgtmCancelRe = regexp.MustCompile(`(?mi)^\/lgtm cancel\r?$`)
 )
 
-// Add or remove LGTM when reviewers comment "/lgtm" or "/lgtm cancel".
-func (ga *GitHubAgent) lgtmComment(ic github.IssueCommentEvent) error {
+type GitHubClient interface {
+	CreateComment(owner, repo string, number int, comment string) error
+	AddLabel(owner, repo string, number int, label string) error
+	RemoveLabel(owner, repo string, number int, label string) error
+}
+
+// HandleIssueComment adds or remove LGTM when reviewers comment "/lgtm" or
+// "/lgtm cancel".
+func HandleIssueComment(gc GitHubClient, ic github.IssueCommentEvent) error {
```

Comment:
Yeah, I think we should use a design closer to what we have in the mungebot, with Munger interface etc